ddtrace/contrib/django/patch.py

```diff
@@ -104,6 +107,20 @@ def patch_conn(django, conn):
                 psycopg_cursor_cls = None
                 Psycopg2TracedCursor = None
 
+    CONN_ATTR_BY_TAG = {
+        net.TARGET_HOST: "HOST",
+        net.TARGET_PORT: "PORT",
+        db.USER: "USER",
+        db.NAME: "NAME",
+    }
```

Comment:
we can probably define this at the module level?

Reply:
nit-pick, non-blocking comment.

Reply:
done